presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -8,16 +8,10 @@ services:
     volumes:
       - ../../../../:/docker/volumes/presto
 
-  hadoop-base:
-    image: 'teradatalabs/cdh5-hive:2'
-    env_file:
-      - ../../../target/classes/presto.env
-    volumes:
-      - ../../../../:/docker/volumes/presto
-
-  hadoop-master-common:
+  hadoop-master:
     extends:
-      service: hadoop-base
+      service: java-8-base
+    image: 'teradatalabs/cdh5-hive:2'
```

Comment:
Why do we need to change the image here to `cdh5-hive`? In non-kerberized environment Presto must be run within the simple `java8` container.